dell_scripts/mountcd.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+#### This script has to mount the iso in the server's virtualmedia and return 0 if operation succeeded, 1 otherwise
+#### Note: Iso image to mount will be received as the first argument ($1)
+
+ISO=${1}
+
+# Disconnect image just in case
+/opt/dell/srvadmin/bin/idracadm7 -r 192.168.1.10 -u root -p calvin remoteimage -d
```

Comment:
We should either drop Dell scripts or make them more production-looking. At least provide the IP address and credentials.

Reply:
Note that this is meant only to illustrate how it would be done. Dell iDracs don't need this as they support Redfish directly as shipped with Metal3. Perhaps a good idea is to mention this in a comment in the files to make it very obvious for anyone reading them and not be tempted to put them to production by misunderstanding their purpose?

Reply:
I force pushed changes to put a disclaimer in the README and in the scripts.

Reply:
Random idea: would there be value in talking to something easier to set up in dev than an actual BMC - say, running virsh commands or ipmitool? Or are we better off using examples that are closer to reality like the idracadm one?

Reply:
IMO I wouldn't move the iDrac examples to use virsh or ipmitool instead. These scripts with iDrac are relatable to potential users and illustrative of the how this can be done. If we want to reinforce it by adding another set of scripts showing virsh or ipmitool that's a nice to have. I don't find it necessary though, since these ones serve the purpose well already.